statics/js/components/topology.js

```diff
@@ -122,6 +135,7 @@ var TopologyComponent = {
       timeRange: [-120, 0],
       collapsed: false,
       topologyFilter: "",
+      topologySelect: "",
```

Comment:
nit: I would not use "topologySelect" as it lead to think that we select something. Here it is just matter of highlighting